src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs

```diff
@@ -434,7 +434,7 @@ private static class CultureInfoMapper
                     {"Lower Sorbian (Germany)", "dsb-DE"},
                     {"Luxembourgish (Luxembourg)", "lb-LU"},
                     {"Macedonian", "mk"},
-                    {"Macedonian (Former Yugoslav Republic of Macedonia)", "mk-MK"},
+                    {"Macedonian (North Macedonia)", "mk-MK"},
```

Comment:
This should be "Republic of North Macedonia"

Reply:
From where you get this info? It should be just North Macedonia.

Reply:
It is optional we can keep it as-is.